conans/client/installer.py

```diff
@@ -227,6 +225,14 @@ def build_package(self, node, keep_build, recorder, remotes):
             return node.pref
 
 
+def remove_folder_raising(folder):
```

Comment:
Make this function _private_, (`_remove_folder_raising`) or move it to `conans.client.tools.files`